keyform/templates/keyform/contact.html

```diff
@@ -0,0 +1,43 @@
+{% extends "keyform/base.html" %}
+{% load i18n %}
+{% load static %}
+{% block css %}
+    {{ block.super }}
+    <link rel='stylesheet' type='text/css' href="{% static 'keyform/css/contact.css' %}"/>
+{% endblock %}
+{% block javascript %}
+    {{ block.super }}
+    <script src='{% static "keyform/js/contact.js" %}'></script>
+{% endblock %}
+
+{% block content %}
+    <h2 class='title'>{% trans 'Contacts' %}</h2>
+    <div class="contact-container">
+        <div class='buildings'>
+            {% for building in buildings %}
+                <div class="building row">
+                    <div class="large-12 columns">
+                        <span class="building-name"> {{ building.name }} </span>
+                    </div>
+                </div>
+                {% for contact in building.contact_set.all %}
+                    <div class="contact row" data-pk='{{ contact.pk }}'>
+                        <a href='{% url "edit_contact" pk=contact.pk %}'>
+                            <div class="info">
+                                <div class='large-4 columns'>
+                                    <span class='contact-info'>{{ contact.name }}</span>
+                                </div>
+                                <div class='large-7 columns end'>
+                                    <span class='contact-info'>{{ contact.email }}</span>
+                                </div>
+                            </div>
+                        </a>
+                        <div class='large-1 columns delete'>X</div>
+                    </div>
+
+                {% endfor %}
+                <a href="{% url 'create_contact' %}"><button class='new'>{% trans 'Create Contact' %}</button></a>
```

Comment:
The URL should be changed to `create-contact` and `button` should be added to the list of classes for the <button> tag.